pom.xml

```diff
@@ -347,26 +347,26 @@
     <surefire-forkcount>8</surefire-forkcount>
 
     <!-- dependency versions -->
-    <slf4j.version>1.7.35</slf4j.version>
-    <audience-annotations.version>0.5.0</audience-annotations.version>
+    <slf4j.version>1.7.36</slf4j.version>
+    <audience-annotations.version>0.12.0</audience-annotations.version>
```

Comment:
Can this go to the newest version: 0.14.1?

Reply:
Thanks, good idea. I tried, but 0.14.1 seems to work only with java11+
(build error: `class file has wrong version 55.0, should be 52.0`)
But I can upgrade to 0.13.0. I see (according to maven central) that 0.13.0 is a widely used version, I upgraded to it.